client/src/components/SignIn/SignIn.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+
+import { loginUser } from './../../authentication';
+
+class Login extends Component {
+    state = {
+        email: '',
+        password: ''
+    }
+    handleInputChange = (event) => {
+        this.setState({
+            [event.target.name]: event.target.value
+        })
+    }
+
+    handleSubmit = (event) => {
+        event.preventDefault();
+        const { email,  password } = this.state;
+        const user = {
+            email,
+            password,
+        }
+        loginUser(user);
+    }
+    render() {
+        console.log('this.props', this.props)
+        return (
+            <div className="container">
+                <h2 style={{marginBottom: '40px', color: 'white'}}>Please sign in</h2>
```

Comment:
It should be better to write styles in the style file, even if they are there temporarily. Especially if this file has already been created.

Reply:
Ok, thanks